Att bytes already set

```
catch.js:77 Error: Att bytes already set
    at Att.setData (chrome-extension://nhlmpmiobabndkmnhocbgomijbcmiikg/js/common/core/att.js:12:23)
    at downloadDataIfNeeded (chrome-extension://nhlmpmiobabndkmnhocbgomijbcmiikg/chrome/elements/attachment.js:154:23)
    at async handleDownloadButtonClicked (chrome-extension://nhlmpmiobabndkmnhocbgomijbcmiikg/chrome/elements/attachment.js:172:21)
    at <async> downloadDataIfNeeded (/extension/chrome/elements/attachment.ts:138:31)

### Catch.reportErr calling stack ###
#     at Function.Catch.formattedStackBlock (chrome-extension://nhlmpmiobabndkmnhocbgomijbcmiikg/js/common/platform/catch.js:140:44)
#     at Function.Catch.reportErr (chrome-extension://nhlmpmiobabndkmnhocbgomijbcmiikg/js/common/platform/catch.js:145:26)
#     at renderErr (chrome-extension://nhlmpmiobabndkmnhocbgomijbcmiikg/chrome/elements/attachment.js:74:23)
#     at handleDownloadButtonClicked (chrome-extension://nhlmpmiobabndkmnhocbgomijbcmiikg/chrome/elements/attachment.js:181:21)
# 
# url: chrome-extension://nhlmpmiobabndkmnhocbgomijbcmiikg/chrome/elements/attachment.htm?frameId=frame_LMnbCjwwlW&msgId=15f7f6780c0c7432&name=node-v6.10.0-linux-x64.tar.xz.pgp&type=application%2Fx-xz&size=9377514&attId=ANGjdJ9MemZv3BdPZQF7cUKxwBFLXfn3s0dAF2j43D5e7P8RQwjfxfDSuCrEvF9pDamAFxqSQ97M9Un7qqlnIUtRmW18phY7saWfeawCBBVL00ANOp6bV5r48zAQXYHTsrTzLIh7oH8jXO8d7pMybV3XZByB8qqkfoaE7BPKgrUO949DGx6hePu-nmUNDCsLkOm6cAWOvmefzSF0m0sW0y5mQDWZrcIFed-ZvvsSemy2rqEDejX72GLnxU8C4DU417KX3FmnbichO_H-2qIegcIBlOoMZz8hct9a4P6b2QrU-qpfl8SbKmVS9K0edik8jXW0Kju1EEG5POabOspETWqpfhrgdsKM_EdcGxc-b7N2G9rmHO36FGIYLhz7pPC_VFHTk_L1saD4drV1rUux&isEncrypted=___cu_true___&acctEmail=flowcrypt.compatibility%40gmail.com&parentTabId=2%3A0
# 
######################
```

https://mail.google.com/mail/u/flowcrypt.compatibility@gmail.com/#sent/WhctKJVjQgXRmNJNwxKZQZnNHdcvNDddfgcgklRrGFKNnvsVqKsDvrgmhPHQXxTnmLBhSbq

--- a/extension/chrome/elements/attachment.ts
+++ b/extension/chrome/elements/attachment.ts
@@ -35,6 +35,7 @@ View.run(class AttachmentDownloadView extends View {
   private button = $('#download');
   private originalButtonHTML: string | undefined;
   private canClickOnAtt: boolean = false;
+  private downloadInProgress = false;
 
   constructor() {
     super();
@@ -82,7 +83,7 @@ View.run(class AttachmentDownloadView extends View {
   setHandlers = () => {
     Ui.event.protect();
     if (this.canClickOnAtt) {
-      this.button.click(this.setHandlerPrevent('double', this.downloadButtonClickedHandler));
+      this.button.click(this.setHandlerPrevent('double', () => this.downloadButtonClickedHandler()));
     }
   }
 
@@ -161,6 +162,10 @@ View.run(class AttachmentDownloadView extends View {
   }
 
   private downloadButtonClickedHandler = async () => {
+    if (this.downloadInProgress) {
+      return;
+    }
+    this.downloadInProgress = true;
     try {
       this.originalButtonHTML = this.button.html();
       this.button.addClass('visible');
@@ -174,6 +179,8 @@ View.run(class AttachmentDownloadView extends View {
       }
     } catch (e) {
       this.renderErr(e);
+    } finally {
+      this.downloadInProgress = false;
     }
   }
 

--- a/extension/chrome/elements/composer/composer-size.ts
+++ b/extension/chrome/elements/composer/composer-size.ts
@@ -51,7 +51,6 @@ export class ComposerSize extends ComposerComponent {
 
   resizeComposeBox = (addExtra: number = 0) => {
     if (this.view.isReplyBox) {
-      console.log('evaluating');
       this.composer.S.cached('input_text').css('max-width', (this.composer.S.cached('body').width()! - 20) + 'px'); // body should always be present
       let minHeight = 0;
       let currentHeight = 0;

